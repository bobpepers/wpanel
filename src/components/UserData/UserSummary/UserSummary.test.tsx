import {shallow} from 'enzyme';
import * as React from 'react';
import {UserSummary, UserSummaryProps} from './index';

const defaults: UserSummaryProps = {
    user: {phones: [], profile: {country: 'US', metadata: { nationality: 'American'}}},
};

describe('UserSummary component', () => {
    const setup = (props: Partial<UserSummaryProps> = {}) =>
        shallow(<UserSummary {...{...defaults, ...props}}/>);

    it('should render', () => {
        const wrapper = setup();
        expect(wrapper).toMatchSnapshot();
        expect(wrapper).toBeDefined();
    });
});
