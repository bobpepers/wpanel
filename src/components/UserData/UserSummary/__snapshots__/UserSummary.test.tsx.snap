// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserSummary component should render 1`] = `
<UserSummaryComponent
  classes={
    Object {
      "gridRow": "UserSummaryComponent-gridRow-1",
      "label": "UserSummaryComponent-label-4",
      "paper": "UserSummaryComponent-paper-3",
      "title": "UserSummaryComponent-title-2",
    }
  }
  user={
    Object {
      "phones": Array [],
      "profile": Object {
        "country": "US",
        "metadata": Object {
          "nationality": "American",
        },
      },
    }
  }
/>
`;
